config/environment.rb

```diff
@@ -3,3 +3,17 @@
 
 # Initialize the Rails application.
 Rails.application.initialize!
+
+ActionMailer::Base.delivery_method = :smtp
+ActionMailer::Base.perform_deliveries = true
+ActionMailer::Base.smtp_settings ={
+  address: 'smtp.gmail.com',
+  port: 465,
+  domain: 'gmail.com',
+  user_name: 'meets9921@gmail.com',
+  password: 'meet2604',
```

Comment:
Move this to credntials.

Reply:
Has been already implemented in gem branch, will be updated when merged.